app/views/tweets/edit.html.erb

```diff
@@ -1,3 +1,10 @@
+<% if @tweet.errors.full_messages.any? %>
+  <h4>Errors</h4>
+  <% @tweet.errors.full_messages.each do |error_message| %>
+    <%= error_message if @tweet.errors.full_messages.first == error_message %> <br />
```

Comment:
@TheJhyde and I agreed that the UX for editing a tweet can be improved in the feature.  Also, we currently allow people to post ridiculously long tweets - see below.  Somewhere down the road, we should stop allowing this.
<img width="987" alt="screen shot 2017-07-11 at 8 50 52 pm" src="https://user-images.githubusercontent.com/5529982/28098256-cf602324-667a-11e7-8ea0-5fa0336f796d.png">